File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    const result = [0, 0];
    const big = Math.max(n, m);
    const small = Math.min(n, m)
    
    for(let i = 1; i < big; i++) {
        if(n % i === 0 && m % i === 0 && result[0] < i) result[0] = i;
    }
    
    let count = 1;
    while(!result[1]) {
        if(big * count % small === 0) result[1] = big * count;
        else count++;
    }
    
    return result;
}

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.c
#include <stdio.h>

int main(void)
{
    int a, b, max, min;
    
    scanf("%d %d", &a, &b);
	if (a > b)
		max = a, min = b;
	else
		max = b, min = a;
    for (int i = max; i > 0; i--)
	{
		if (max % i == 0 && min % i == 0)
		{
			printf("%d", i);
			break ;
		}
	}
	for (int i = 1; i != 0; i++)
	{
		if ((min * i) % max == 0)
		{
			printf(" %d", min * i);
			break ;
		}
	}
    return (0);
}